SPECS/python-pytest-cov/SOURCES/32d8c0665ea044f9682a89633d75442d1a4d2dc4.patch

```diff
@@ -0,0 +1,39 @@
+From 32d8c0665ea044f9682a89633d75442d1a4d2dc4 Mon Sep 17 00:00:00 2001
```

Comment:
>From [](http://example.com/codeflow?start=0&length=4)

Nit: Can we move this out of the SOURCES folder and rename this to something more descriptive?